src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/CommandLineApplication.cs

```diff
@@ -332,6 +332,10 @@ public void ShowHelp(string commandName = null)
             else
             {
                 target = Commands.SingleOrDefault(cmd => string.Equals(cmd.Name, commandName, StringComparison.OrdinalIgnoreCase));
+                if (target == null)
+                {
+                    throw new Exception(string.Format("TODO: Error: unexpected value '{0}' for option 'help'", commandName));
```

Comment:
Sent from my HTC

----- Reply message -----
From: "Murat Girgin" <notifications@github.com>
To: "aspnet/dnx" <dnx@noreply.github.com>
Cc: "Brennan Conroy" <brecon@microsoft.com>
Subject: [dnx] Fix null-ref on bad arg to help (#1893)
Date: Mon, May 18, 2015 7:49 PM


In src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/CommandLineApplication.cs<https://github.com/aspnet/dnx/pull/1893#discussion_r30565661>:

> @@ -332,6 +332,10 @@ public void ShowHelp(string commandName = null)
>              else
>              {
>                  target = Commands.SingleOrDefault(cmd => string.Equals(cmd.Name, commandName, StringComparison.OrdinalIgnoreCase));
> +                if (target == null)
> +                {
> +                    throw new Exception(string.Format("TODO: Error: unexpected value '{0}' for option 'help'", commandName));


String interpolation

-
Reply to this email directly or view it on GitHub<https://github.com/aspnet/dnx/pull/1893/files#r30565661>.